utils/controleval.py

```diff
@@ -162,22 +231,27 @@ def print_to_json(ctrls, product, id, level):
     data["addressed_controls"]["all"] = get_id_array(ctrls)
     data["addressed_controls"]["applicable"] = get_id_array(applicablelist)
     data["addressed_controls"]["assessed"] = get_id_array(assessedlist)
+    data["addressed_controls"]["notassessed"] = get_id_array(applicablelist - assessedlist)
+    data["addressed_controls"]["automated"] = get_id_array(
+        ctrllist[controls.Status.AUTOMATED])
+    data["addressed_controls"]["doesnotmeet"] = get_id_array(
+        ctrllist[controls.Status.DOES_NOT_MEET])
+    data["addressed_controls"]["documentation"] = get_id_array(
```

Comment:
Think if it would be better to transofrm this to a loop.

Reply:
Hum, interesting. I will send another commit to fix the CodeClimate founds and will already see how to make this function even better. Thanks

Reply:
Actually, as informed in the PR description, I also want to refactor this function but I was conservative with it in this PR. I will refactor it in a different PR so I the authors can also review it separated. The idea is to use the new functions and apply the same logic used to generate the "human" output to the json output.